src/stem/urls.py

```diff
@@ -33,4 +35,6 @@
     path('', RedirectView.as_view(url='api/v1/')),
     path('admin/', admin.site.urls),
     path('api/v1/', include(router.urls)),
+    path(r'api-token-auth/', obtain_jwt_token),
+    path(r'api-token-refresh/', refresh_jwt_token),
```

Comment:
I don't believe it breaks anything but I might be wrong. Could we match convention on the other URL routes by using `api/token/auth` and `api/token/refresh` or something? Also, I'm not sure we need to make these raw strings as the others work without that setup.

Reply:
Yes, sure thing. 
All I want is to make JWT work so that we can mould to whatever we want. Down the road, coming sprint has a lot of things to implement around this.